libmcount/dynamic.c

```diff
@@ -110,6 +115,18 @@ struct mcount_orig_insn *mcount_save_code(struct mcount_disasm_info *info,
 		memcpy(orig->orig, (void *)info->addr, info->orig_size);
 	}
 
+	index = info->copy_size + jmp_size;
+	/* patch conditional jumps instructions */
+	for(i = 0; i < info->nbr_cond_branch; i++){
+		memcpy(cond_branch_trampoline + JMP_INSN_SIZE, &info->cond_info[i].jmp_addr, sizeof(info->cond_info[i].jmp_addr));
+		memcpy(orig->insn + index, cond_branch_trampoline, sizeof(cond_branch_trampoline));
+
+		uint8_t target = (unsigned long)info->insns + index - ((unsigned long)info->insns + info->cond_info[i].insn_index + 2);
+		info->insns[info->cond_info[i].insn_index + 1] = target;
```

Comment:
Please try to follow the coding style.  The lines are too long, please consider using a local variable.  Also it seems not possible that target is great than 128 but it would be better to add a check to verify that.